File: week6/sentimental-mario-less/mario.py
# TODO
from cs50 import get_int
def getinput ():
    while True :
        h = get_int("height :")
        if h>0 and h<9:
            return h


def main():
    h = (getinput())
    t = 1
    for i in range(h):
        print(" "*(h-1) ,end='')
        h = h-1
        print("#"*t)
        t = t+1
main()

File: week1/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // declaring the height
    int n;
    // getting the number by forcing the user to type a number between 1 and 8
    do
    {
        n = get_int("height = ");

    }
    while ((n > 8) || (n < 1));

    // the pyramid
    for (int i = 0; i < n; i++)
    {
        for (int j = n; j > i + 1; j--)
        {
            printf(" ");
        }
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }
        printf("\n");
    }

}